### ISSUE TITLE
consolidate route table configuration

### ISSUE THREAD

**Issue by wyattwalter** [department-of-veterans-affairs/va.gov-team#3674]
## User Story or Problem Statement
While doing the cutover for the Transit Gateway work, I noticed that we had separate route tables in AWS for some reason for each AZ, but they all seem to be identical. How might we consolidate these into a single entity so we don't have to update multiple (and potentially cause conflicts between them) per VPC?

## Goal
De-duplicated route table configuration.

## Objectives or Key Results this is meant to further
- Enable agile VFS delivery of performant applications

## Acceptance Criteria
- [x] Route tables that are copies of one another should be consolidated into a single entity
- [x] Dev
- [x] Utility
- [x] Staging 
- [x] Prod

## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [x] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `triage`, `tools-improvements`)
- [x] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `call center`, `research`, `accessibility`, `content`)
- [x] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by wyattwalter** [comment]
@ricetj do you have a good epic to put this under? It's a small-ish cleanup item.

**Comment by wyattwalter** [comment]
This is true for all of our environments, but there's a route table for each AZ like this:

https://github.com/department-of-veterans-affairs/devops/blob/master/terraform/environments/dsva-vagov-dev/vpc.tf#L136

They're all the same, and could be cleaned up to take out quite a bit of code in that file.

**Comment by ricetj** [comment]
@benniemosher I know you are blocked here, but could you leave a small note in case someone else needs to pick this up while you are on support? Thank you.

**Comment by benniemosher** [comment]
@ricetj There are a bunch of terraform move steps that need to be done. If this becomes unblocked while I am on support, my vote is we either wait until I am not, or I just take the 15 minutes to do the commands really fast during the day. This is not critical work and can hang out for awhile, it is in its own PR. 

Basically here are the steps that I do:

- `git fetch; git rebase origin/master`
- `tf init - upgrade`
- `tf plan`
- I then copy the resources changed between the line breaks `---` and then make commands that mv any `rt_1a` resources into the `rt_1` resources. At the time of this comment this is the list:

```
tf state mv aws_route.rt_1a_appeals_vpc aws_route.rt_1_appeals_vpc
tf state mv aws_route.rt_1a_dvp_prod aws_route.rt_1_dvp_prod
tf state mv aws_route.rt_1a_old_vpc aws_route.rt_1_old_vpc
tf state mv aws_route.rt_1a_tgw_default aws_route.rt_1_tgw_default
tf state mv aws_route.rt_1a_utility aws_route.rt_1_utility
tf state mv aws_route.rt_1a_vautility aws_route.rt_1_vautility
tf state mv aws_route_table.rt_1a aws_route_table.rt_1
tf state mv aws_route_table_association.rt_1a aws_route_table_association.rt_1_subnet_1a
tf state mv aws_route_table_association.rt_1b aws_route_table_association.rt_1_subnet_1b
tf state mv aws_route_table_association.rt_1c aws_route_table_association.rt_1_subnet_1c
``` 

- I then will run a `tf plan` to verify that I it is in the state I want, if I missed any will run one off commands to get it cleaned up.
- Then will run a `tf apply` and it should go through green without any extra intervention in the AWS console.

**Comment by ricetj** [comment]
Done and working
